src/screens/ActivitiesPage.js

```diff
@@ -1,8 +1,56 @@
+import { Grid, Typography } from '@mui/material';
+import SelCard from '../components/SelCard';
+
 const ActivitiesPage = () => {
   return (
-      <>
-      This is the Activities page. 
-      </>
+    <Grid
+      container
+      direction="column"
+      justifyContent="center"
+      alignItems="center"
+      spacing={2}
+      p={8}
+    >
+      <Grid item xs={12}>
+        <Typography variant="h2" align='center'>SEL Activities by Skill Category</Typography>
+      </Grid>
+
+      <Grid item m={1}>
+        <SelCard
+          title={'Responsible Decision Making'}
+          description={'Understanding how our decisions can impact others and ourselves'} 
+          progress={72} />
+      </Grid>
```

Comment:
Make sure to link these to their respective pages as well - check how I did it in Card component.